manager/container.go

```diff
@@ -270,6 +273,44 @@ func (c *containerData) updateLoad(newLoad uint64) {
 	}
 }
 
+func (c *containerData) updateSubcontainerStats(stats *info.ContainerStats, path string) (*info.ContainerStats, error) {
```

Comment:
Sure, and I'll change it to be recursive. That seems like it would be more efficient, so that we're not always checking the sub-sub containers of a given container.